### PR TITLE
Feature #3: Created main menu.

### PR DESCRIPTION
**Pull request by nukesz**

### CODE REVIEW

File: core/src/com/nukesz/game/MenuScreen.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        table.add(options).fillX().uniformX();
+        table.row();
+        table.add(exit).fillX().uniformX();
+
+        play.addListener(new ChangeListener() {
+            @Override
+            public void changed(ChangeEvent event, Actor actor) {
+                game.setScreen(new GameScreen(game));
+            }
+        });
```
Comment (palocziarpad) [inline core/src/com/nukesz/game/MenuScreen.java]: If possible, please rewrite this and the folliwing to addlistener to lambda expression.

File: core/src/com/nukesz/game/MenuScreen.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+        options.addListener(new ChangeListener() {
+            @Override
+            public void changed(ChangeEvent event, Actor actor) {
+                // TODO
```
Comment (palocziarpad) [inline core/src/com/nukesz/game/MenuScreen.java]: Please do the todo or remove the comment

File: core/src/com/nukesz/game/MenuScreen.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        });
+
+    }
+
+    @Override
+    public void render(float delta) {
+        Gdx.gl.glClearColor(0f, 0f, 0f, 1);
+        Gdx.gl.glClear(GL20.GL_COLOR_BUFFER_BIT);
+
+        stage.act(Math.min(Gdx.graphics.getDeltaTime(), 1 / 30f));
```
Comment (palocziarpad) [inline core/src/com/nukesz/game/MenuScreen.java]: Please create a constant for " 1 / 30f" expression.

File: core/src/com/nukesz/game/MenuScreen.java (hunk 1 above)
Comment (nukesz) [reply core/src/com/nukesz/game/MenuScreen.java]: Android does not support java 8. If we want to target it, then we can't use lamba.

Comment (palocziarpad) [reply core/src/com/nukesz/game/MenuScreen.java]: I see, then please ignore this comment.

File: core/src/com/nukesz/game/MenuScreen.java (hunk 3 above)
Comment (nukesz) [reply core/src/com/nukesz/game/MenuScreen.java]: changed

File: core/src/com/nukesz/game/MenuScreen.java (hunk 2 above)
Comment (nukesz) [reply core/src/com/nukesz/game/MenuScreen.java]: done

**Review by palocziarpad — APPROVED** [review]
Good job, the pictures are looking nice. 
I have done manual testing it works as intended.

**Pull request merged**